config/install/migrate_plus.migration.joomla_node.yml

```diff
@@ -19,6 +19,17 @@ process:
   sticky:
     plugin: default_value
     default_value: 0
+  langcode:
+    plugin: default_value
+    source: language
+    default_value: "und"
+  status: state
```

Comment:
Drupal's node status field expects a Boolean value. If we need to accommodate additional states, we need to configure Workbench to match the four options, and implement prepareRow in the JoomlaNode class to map Joomla states (1,0,-1,-2) to Drupal workbench states ('draft', 'review', 'published', 'archive'."

This requirement should be managed in a separate ticket because it assumes hefty setup has been managed already.